utils/modules/geom_utils.py

```diff
@@ -74,3 +74,22 @@ def mean(verts):
 
 def is_reasonably_opposite(n, normal_one):
     return dot_v3v3(normalized(n), normalized(normal_one)) < 0.0
+
+
+def ptInTriang(p_test, p0, p1, p2):
```

Comment:
`def pt_in_triangle` this module has lowercase function names with words separated by `_` , as is common practice in Python code.